roles/sriov-dp-install/charts/sriov-net-dp/templates/configMap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-sriovdp-config
+  namespace: {{ .Values.namespace }}
+data:
+  config.json: |
+    {
+        "resourceList": [{
+                "resourceName": "intel_sriov_netdevice",
+                "selectors": {
+                    "vendors": ["8086"],
+                    "devices": ["154c", "10ed"],
+                    "drivers": ["i40evf", "ixgbevf"]
+                }
+            },
+            {
+                "resourceName": "intel_sriov_dpdk",
+                "selectors": {
+                    "vendors": ["8086"],
+                    "devices": ["154c", "10ed"],
+                    "drivers": ["vfio-pci"]
```

Comment:
@ahalim-intel should I append `igb_uio` here?

Reply:
No, there's no need for that. The igb_uio is not considered secure for multi-tenant scenario and it's a good idea to avoid that. Let's just keep vfio-pci in the example config. The igb_uio is supported but not promoted.